README.md

```diff
@@ -4,7 +4,8 @@
 [![codecov](https://codecov.io/gh/akuity/kargo/branch/main/graph/badge.svg?token=FGUq4netA6)](https://codecov.io/gh/akuity/kargo)
 [![Netlify Status](https://api.netlify.com/api/v1/badges/c4e62c70-e07d-4ac6-bcf8-d7003c936d19/deploy-status)](https://app.netlify.com/sites/docs-kargo-akuity-io/deploys)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)
-[![slack](https://img.shields.io/badge/slack-kargo-brightgreen.svg?logo=slack)](https://akuity-community.slack.com/messages/C05BXBX7695)
+![Discord](https://img.shields.io/discord/:1138942074998235187)
```

Comment:
```suggestion
[![Discord](https://img.shields.io/discord/1138942074998235187)](https://discord.gg/dHJBZw6ewT)
```
With the colon, the link doesn't work for me; without it does. Despite the docs showing the colon.
![Xnapper-2023-09-12-11 11 56](https://github.com/akuity/kargo/assets/4213435/8147b90d-14e8-4c5c-a55a-1cf9f14f821f)
![Xnapper-2023-09-12-11 12 09](https://github.com/akuity/kargo/assets/4213435/414c9b44-ee94-4a22-b1be-c603a240b070)
Also, this could be a link to right?

Reply:
@krancour it's the colon causing it ^

Reply:
Yep. That's it.